packages/doenetml-worker-rust/lib-doenetml-core/src/core/props/prop_value.rs

```diff
@@ -56,15 +58,17 @@ pub mod prop_type {
     //! so that they can be used in macros.
 
     use super::*;
-    use crate::state::types::element_refs;
+    use crate::state::types::{content_refs, element_refs};
 
     pub type String = Rc<std::string::String>;
     pub type Number = f64;
     pub type Integer = i64;
     pub type Boolean = bool;
     pub type Math = Rc<MathExpr>;
-    pub type ElementRefs = Rc<element_refs::ElementRefs>;
-    pub type GraphNodes = Rc<Vec<GraphNode>>;
+    pub type ComponentRef = Option<element_refs::ElementRef>;
```

Comment:
Having both terms `ComponentRef` and `ElementRef` is confusing. They seem to mean the same thing: a reference to a component via its `ComponentIdx`. I don't have a strong preference, but we should pick one and standardize around it. I guess with `ComponentIdx`, we seem to use `component` more than `element`.

`ComponentRef/ElementRef` is in contrast to `ContentRef` which is either a component or a string. I'm assuming that is the only difference. In that case, I'm confused why `prop_type:ComponentRef` is an option while `prop_type:ContentRef` is not. I'm going to assume it is due to a use case that we've had with `ComponentRef/ElementRef` that we haven't had with `ContentRef`.